gateway-service-impl/src/main/java/org/hypertrace/gateway/service/common/converters/StringToAttributeKindConverter.java

```diff
@@ -86,12 +87,13 @@ public Value doConvert(String value, AttributeKind attributeKind, Value.Builder
       return List.of();
     }
 
+    // Check if the string is already in a list format.
     try {
       return objectMapper.readValue(jsonString, LIST_TYPE_REFERENCE);
-    } catch (IOException e) {
-      LOGGER.warn("Unable to read List JSON String data from: {}. Setting data as empty list instead. With error:", jsonString, e);
+    } catch (JsonProcessingException e) {
```

Comment:
Cleaner long term fix is to modify the EDS to return list of strings as a proper attribute list in the response.